docs/src/basics.md

```diff
@@ -0,0 +1,47 @@
+# How does it work  ?
+
+Boscia focuses on mixed-integer convex problems in which the nonlinear constraints and objectives are convex and presents a new algorithmic framework for solving these problems that exploit recent advances in so-called Frank-Wolfe (FW) or Conditional Gradient (CG) methods. The problem class we consider is of the type: 
+$$
+\min ( f(x) ) \text{ where } x \in \mathbb{R}^n, \, x \in X
+$$
+
+where \( X \) is a compact nonconvex set admitting a boundable linear minimization oracle (LMO), i.e., a set over which optimizing a linear function can be done efficiently (comparatively to the original problem), even when bounds are added or modified. Formally, we consider we have access to an oracle taking new bounds \( (l, u) \) and a direction \( d \):
+
+Given : 
+$$
+\( (l, u, d) \in \mathbb{R}^n \times \mathbb{R}^n \times \mathbb{R}^n \),  \( \arg\min_{v \in \mathbb{R}^n} \langle v, d \rangle \) 
+ \text {subject to}    \( v \in X \cap [l, u] \).
+$$
+
+
+## BPCG 
```

Comment:
BPCG is the default but not the only variant that can be used. I would move this more back.

Reply:
what to do in this

Reply:
Move it further down the page and state that it is used as the default.